StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml

```diff
@@ -108,77 +124,68 @@
       <min value="0" />
       <max value="1" />
       <type>
         <code value="Extension" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-ControlledDrug" />
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMControlledDrug" />
       </type>
-      <mustSupport value="true" />
+      <isModifier value="false" />
     </element>
     <element id="MedicationRequest.extension:dispensingInformation">
       <path value="MedicationRequest.extension" />
       <sliceName value="dispensingInformation" />
-      <short value="For EPS use only" />
-      <definition value="This extension is used in `continuous-repeat-dispensing` prescriptions sent by EPS to dispensing systems to convey information on prior dispensed medications.&#xD;&#xA;&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;extension&quot;:  [&#xD;&#xA;                {&#xD;&#xA;                    &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-EPS-DispensingInformation&quot;,&#xD;&#xA;                    &quot;extension&quot;:  [&#xD;&#xA;                        {&#xD;&#xA;                            &quot;url&quot;: &quot;dispenseStatus&quot;,&#xD;&#xA;                            &quot;valueCoding&quot;: {&#xD;&#xA;                                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/medicationdispense-type&quot;,&#xD;&#xA;                                &quot;code&quot;: &quot;0001&quot;,&#xD;&#xA;                                &quot;display&quot;: &quot;Item fully dispensed&quot;&#xD;&#xA;                            }&#xD;&#xA;                        },&#xD;&#xA;                        {&#xD;&#xA;                            &quot;url&quot;: &quot;dateLastDispensed&quot;,&#xD;&#xA;                            &quot;valueDateTime&quot;: &quot;2018-04-22T09:57:03+00:00&quot;&#xD;&#xA;                        }&#xD;&#xA;                    ]&#xD;&#xA;                }&#xD;&#xA;            ],&#xD;&#xA;```" />
       <min value="0" />
       <max value="1" />
       <type>
         <code value="Extension" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-EPS-DispensingInformation" />
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-EPSDispensingInformation" />
       </type>
-      <mustSupport value="true" />
+      <isModifier value="false" />
+    </element>
+    <element id="MedicationRequest.extension:NHSBSAPrescriptionType">
+      <path value="MedicationRequest.extension" />
+      <sliceName value="NHSBSAPrescriptionType" />
+      <min value="1" />
+      <max value="1" />
```

Comment:
Cardinality max to set to 1 but previously the max was not known so presumably would be * by default. Needs discussion.